test/Microsoft.OpenApi.CSharpAnnotations.DocumentGeneration.Tests/OpenApiDocumentGeneratorTests/OpenApiDocumentGeneratorTest.cs

```diff
@@ -1223,6 +1251,62 @@ public void NoOperationsToParseShouldReturnEmptyDocument(string openApiDocumentV
                 .BeEquivalentTo(openApiStringReader.Read(expectedDocument, out var _));
         }
 
+        [Theory]
+        [MemberData(nameof(GetTestCasesForPassingDescriptionShouldReturnCorrectDocument))]
+        public void PassingDescriptionShouldReturnCorrectDocument(
+            string testCaseName,
+            IList<string> inputXmlFiles,
+            IList<string> inputBinaryFiles,
+            string descriptionFile,
+            string expectedJsonFile)
+        {
+            _output.WriteLine(testCaseName);
+
+            var documents = new List<XDocument>();
+
+            documents.AddRange(inputXmlFiles.Select(XDocument.Load));
+
+            var input = new OpenApiGeneratorConfig(
+                documents,
+                inputBinaryFiles,
+                "1",
+                FilterSetVersion.V1)
+            {
+                OpenApiInfoDescription = File.ReadAllText(descriptionFile)
```

Comment:
>          OpenApiInfoDescription = File.ReadAllText(descriptionFile) [](start = 6, length = 68)

Add constructor and use that instead.

Reply:
These are optional settings do not think we need constructor for each one of these.

---
In reply to: [221388517](https://github.com/Microsoft/OpenAPI.NET.CSharpAnnotations/pull/158#discussion_r221388517) [](ancestors = 221388517)